.pylintrc

```
@@ -562,3 +562,11 @@ known-third-party=enchant
 # Exceptions that will emit a warning when being caught. Defaults to
 # "Exception".
 overgeneral-exceptions=Exception
+
+
+[QUOTES]
+
+# Settings for pylint-quotes to enforce the usage of double quotes for strings.
+string-quote=double
```

[Consensus] After a quick glance at the documentation, I believe this should be 
```suggestion
string-quote=double-avoid-escape
```
to have it enforce single quotes if it would reduce escaping